File: problems/100/Main.java
import java.io.*;
import java.util.*;
import java.lang.*;

//--SPLIT--

//-class Solution {
	// X and Y co-ordinates of the points in order.
	// Each point is represented by (X.get(i), Y.get(i))
//-	public int coverPoints(ArrayList<Integer> X, ArrayList<Integer> Y) {
//-	}
//-}



//--SPLIT--
class Solution {
	// X and Y co-ordinates of the points in order.
	// Each point is represented by (X.get(i), Y.get(i))
	public int coverPoints(ArrayList<Integer> X, ArrayList<Integer> Y) {
		int result = 0;
		for (int i = 1; i < X.size(); i++) {
			int a = Math.abs(X.get(i) - X.get(i - 1));
			int b = Math.abs(Y.get(i) - Y.get(i - 1));
			result += Math.abs(b - a);
			result += (a > b) ? b : a;
		}
		return result;
	}
}




//--SPLIT--
public class Main {
	public static void main(String[] args) throws Exception {
		BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
		int cases = Integer.parseInt(reader.readLine());
		for (int i = 1; i <= cases; i++) {
			Solution solution = new Solution();
			int n = Integer.parseInt(reader.readLine());
			ArrayList<Integer> X = new ArrayList<Integer>();
			String[] input = reader.readLine().split(" ");
			for (int j = 0; j < n; j++) {
				X.add(Integer.parseInt(input[j]));
			}
			ArrayList<Integer> Y = new ArrayList<Integer>();
			input = reader.readLine().split(" ");
			for (int j = 0; j < n; j++) {
				Y.add(Integer.parseInt(input[j]));
			}
			System.out.println(solution.coverPoints(X, Y));
		}
	}
}

File: problems/100/Main.py
import math

#--SPLIT--
#-class Solution:
	# @param X : list of integers
	# @param Y : list of integers
	# Points are represented by (X[i], Y[i])
	# @return an integer
#-	def coverPoints(self, X, Y):

#--SPLIT--
class Solution:
	def coverPoints(self, X, Y):
		return 0


#--SPLIT--

if __name__ == "__main__":
    test_cases = int(raw_input())
    for i in range(test_cases):
        n = int(raw_input())
        X = [int(l) for l in raw_input().split(" ")]
        Y = [int(l) for l in raw_input().split(" ")]
        solution = Solution()
        print(solution.coverPoints(X, Y))